static/js/lightbox_canvas.js

```diff
@@ -241,7 +241,7 @@ export class LightboxCanvas {
         this.context = this.canvas.getContext("2d");
 
         this.meta.image = new Image();
-        this.meta.image.src = this.canvas.getAttribute("data-src");
+        this.meta.image.src = this.canvas.dataset.src;
```

Comment:
(In case the [previous discussion](https://github.com/zulip/zulip/pull/13569#issuecomment-570745572) about the jQuery `.data()` setter failing to synchronize with DOM `.getAttribute()` comes up again: there is no difference between DOM `.getAttribute()` and DOM `.dataset` for that purpose. DOM `.getAttribute()` and DOM `dataset` are always synchronized. The jQuery `.data()` getter is synchronized too unless the jQuery `.data()` setter has been called.)

Reply:
I think this is fine; in any case we're hoping to delete this module in the next few days anyway.